TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+# Transaction Author Agreement V2
+```
+                             `-::-`                      
+                      ```   `//////`   ```               
+                    .////:` `//////` `:////.             
+                    //////-  `-::-`  -//////             
+                    .:////`    --    `////:.             
+                ....   ` .:`   --   `:. `   ....         
+              `/////:     `:.  ..  .:`     :////:`       
+              .//////-.`    .  ``  .    `.-//////.       
+               .:::````.---  -////-  ---.```::::.        
+                            .//////.                     
+               .-::-` `.---  -////-  ---.` `-::-.        
+              .//////-.`    `  ``  .    `.-//////.       
+              `/////:`    `:.  ..  .:`    `://///`       
+                .--.     .:`   --   `:.     .--.         
+                    .:////`    --    `////:.             
+                    //////-  `.::.`  -//////             
+                    .////:` `//////` `:////.             
+                      ```   `//////`   ```               
+                             `-::-`                      
+                                                                    
+ ssSSSSSss                                         II               
+sS"     "Ss                                        ""               
+SS,                                                                 
+`SSsssss,     ,ooOOOoo,   Vv       vV  Rr,rRRRrr,  II  Nn,nNNNnn,   
+  `"""""Ss,  oO"     "Oo  `Vv     vV'  RRR'   "RR  II  NNN'   `"Nn  
+        `Ss  OO       OO   `Vv   vv'   RR          II  NN       NN  
+SSs     SSS  "Oo,   ,oO"    `Vv,vV'    RR          II  NN       NN  
+ "SSSSSSS"    `"OoooO"'       "V"      RR          II  NN       NN  
+```
+[https://sovrin.org/](https://sovrin.org/)
+## **Summary:**
+* This summary is provided to help you understand your obligations when writing to the Sovrin Ledger Networks-it does not have any legal effect or replace the full legal text of the agreement provided below it.
+    * This agreement grants you permission to write data to the Sovrin Ledger Networks under certain terms and conditions.
+    * You represent and warrant that the data you are writing does not violate any applicable laws or infringe the rights of any other party.
+    * You understand the data you are writing is public and permanent and there can be no guarantee of erasure. This includes public keys and payment addresses.
+    * If it is determined that the data you wrote violated this agreement, the operators of the network can take steps to block it from public access.
+    * The Sovrin Foundation makes no promises about the reliability or correctness of the data being stored on the Sovrin Ledger Networks or the operation of the Sovrin Ledger Networks.
+~--~
```

Comment:
The lines containing `~--~` are not valid markdown; is this meant to be a horizontal line, separating sections? In either case, I think the header of the next section provides enough of a split that having a horizontal line feels unnecessary to me.